pkg/volume/util/subpath/subpath_windows.go

```diff
@@ -45,30 +45,59 @@ func NewNSEnter(mounter mount.Interface, ne *nsenter.Nsenter, rootDir string) In
 }
 
 // evalPath returns the path name after the evaluation of any symbolic links.
-// If the path after evaluation starts with Volume or \??\Volume, it means that it was a symlink from
-// volume (represented by volumeID) to the given path. In this case, the given path is returned.
+// If the path after evaluation is a device path or network connection, the original path is returned
 func evalPath(path string) (linkedPath string, err error) {
-	cmd := fmt.Sprintf("Get-Item -Path %s | Select-Object -ExpandProperty Target", path)
+	// The following is a workaround solution for issue https://github.com/kubernetes/kubernetes/issues/93759
+	upperpath := path
+	base := ""
+	for {
+		if isDriveLetterorEmptyPath(upperpath) {
+			klog.V(4).Infof("Path '%s' is not a symlink, return its original form.", path)
+			return path, nil
+		}
+		cmd := fmt.Sprintf("(Get-Item -Path %s).Attributes", upperpath)
+		output, err := exec.Command("powershell", "/c", cmd).CombinedOutput()
+		if err != nil {
+			return "", err
+		}
+		if strings.Contains(string(output), "ReparsePoint") {
```

Comment:
Can you add a comment giving an example of this string in some output? And explain what it means?

Reply:
added. PTAL

Reply:
Some docs here seem to indicate that ReparsePoint is only supported by NTFS and also there are multiple kinds of ReparsePoints that are clarified in a fs-specific tag.

https://en.wikipedia.org/wiki/NTFS_reparse_point
https://docs.microsoft.com/en-us/windows/win32/fileio/reparse-points

cc @andyzhangx to help review and see if this is the correct way to identify a symlink.

Reply:
https://docs.microsoft.com/en-us/windows/win32/fileio/determining-whether-a-directory-is-a-volume-mount-point
from my understanding, reparsepoint can be used to check symlink in windows.

Reply:
From that page, it sounds like you also have to parse "IO_REPARSE_TAG_SYMLINK"

Reply:
The list of attributes for a file/dir is follows. I tried all possible options with mklink to create different types of links (https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/mklink), the attribute all include ReparsePoint

ReadOnly
Hidden
System
Directory
Archive
Device
Normal
Temporary
SparseFile
ReparsePoint
Compressed
Offline
NotContentIndexed
Encrypted

Reply:
thanks! I think both approach should work. LinkType seems a new attributed added PowerShell v5.0?